### ISSUE TITLE
Log model that has both built-in scalers and stochastic action output (mu, log_std)

### ISSUE THREAD

**Issue by kostas-maersk** [takuseno/d3rlpy#364]
I really like d3rlpy especially because it implements offline algorithms. My use case is that I'm logging my fitted models (using IQL and continuous actions) to MLFlow. This works, but when logging the model I unfortunately need to choose between two features that I both like! The features I get depend on how I serialise the the fitted model.

1. If I log the model using `model.save_policy('policy.pt')` and load with `torch.jit.load`, then the policy function  automatically scales observations and unscales actions, given that I trained the model (IQL) with scalers from d3rlpy.preprocessing. This is great!
2. If I log the PyTorch module `model.impl.policy`, and load with `mlflow.pytorch.load_model`, then the policy function returns action output with `mu` and `log_std`, which I can use to explore on inference. This is also great!

I would like a convenient way to serialise the policy to get *both of these features* at the same time.
- Solution 1: `model.impl.policy` gets the scalers
- Solution 2: `model.save_policy` creates policy that returns `mu` and `log_std`
- Solution 3: something even better

Maybe it's already possible to do that in current version?

**Comment by takuseno** [comment]
@kostas-maersk Thanks for the issue. Let me check your desired behavior. It sounded like you want to have an access to `mu` and `log_std`, but you want to turn on preprocessing/postprocessing features at the same time. 

It's a shame, but this could be the simplest solution I can propose now:
- 1. Normalize your dataset before you pass it to d3rlpy. You can manually use `d3rlpy.preprocessing` to do this if you want.
- 2. Train your policy with the normalized dataset without giving `d3rlpy.preprocessing` to models during training.
- 3. Use `model.impl.policy`.

Alternatively, you can load `d3` model file that includes all information (e.g. model, preprocessors), which could be difficult to incorporate with mlflow though....

**Comment by kostas-maersk** [comment]
Thank you for your answer Seno-san. Your suggestion resembles one of the things I have tried, namely either manual scaling or manual exploration. 

For now, I deemed the built-in preprocessing more important than using log_std for exploration so I have manually implemented exploration.

I'll see if I can get d3 and MLFlow to work too.
